main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/SdiWorkspaceWindow.cs

```diff
@@ -359,42 +359,38 @@ static void DeepGrabFocus (Gtk.Widget widget)
 			}
 		}
 		
-		static IEnumerable<Gtk.Widget> GetFocussableWidgets (Gtk.Widget widget)
+		static IEnumerable<Gtk.Widget> GetFocusableWidgets (Gtk.Widget widget)
 		{
-			var c = widget as Container;
-
 			if (widget.CanFocus) {
 				yield return widget;
 			}
 
-			if (c != null) {
-				foreach (var f in c.FocusChain.SelectMany (GetFocussableWidgets).Where (y => y != null)) {
+			if (widget is Container c) {
+				foreach (var f in c.FocusChain.SelectMany (GetFocusableWidgets).Where (y => y != null)) {
```

Comment:
```suggestion
				foreach (var f in c.FocusChain.SelectMany (x => GetFocusableWidgets (x))) {
```

I don't see how we can return null from this. Also, Roslyn is better at generating code for the lambda variant, rather than method group, avoiding allocations by using a cached lambda.